### PR TITLE
Styling LoginScreen

### PR DESCRIPTION
**Pull request by fervilla96**
Created the css file for LoginScreen

### CODE REVIEW

File: src/gui/css/loginScreen.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.AppName {
```
Comment (AlexisGlez) [inline src/gui/css/loginScreen.css]: is not a good practice to use uppercase letters in CSS, please change them all to kebab case:
`.AppName` --> `.app-name`

File: src/gui/fxml/LoginScreen.fxml [hunk 2]
```diff
@@ -5,15 +5,16 @@
...
 
 <AnchorPane xmlns="http://javafx.com/javafx/8.0.111" xmlns:fx="http://javafx.com/fxml/1">
    <children>
       <Pane prefHeight="235.0" prefWidth="240.0">
          <children>
-            <TextField layoutX="35.0" layoutY="84.0" promptText="Nombre" />
-            <ImageView fitHeight="57.0" fitWidth="200.0" layoutX="20.0" layoutY="14.0" pickOnBounds="true" preserveRatio="true" />
-            <TextField layoutX="35.0" layoutY="134.0" promptText="Contraseña" />
-            <Button layoutX="35.0" layoutY="184.0" mnemonicParsing="false" prefHeight="26.0" prefWidth="170.0" text="Log In" />
+            <TextField layoutX="35.0" layoutY="84.0" promptText="Nombre" class="inputText"/>
```
Comment (AlexisGlez) [inline src/gui/fxml/LoginScreen.fxml]: Same here:
`class="inputText"` --> `class="input-text"`

**Review by AlexisGlez — CHANGES_REQUESTED** [review]

File: src/gui/css/loginScreen.css (hunk 1 above)
Comment (fervilla96) [reply src/gui/css/loginScreen.css]: okok, thanks (:

**Comment by AlexisGlez** [comment]
I'm seeing that the files names are loginScreen...
Please rename the CSS file from `loginScreen.css` to just `login`

And the FXML file from `LoginScreen.fxml` to `Login.fxml`

**Review by AlexisGlez — APPROVED** [review]
LGTM 💯

**Pull request merged**